templates/paragraph_row.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
+/**
+ * @file paragraphs_row.html.twig
+ * Default theme implementation to present Paragraphs Row data.
+ *
+ * This template is used when viewing Paragraphs Row pages.
+ *
+ *
+ * Available variables:
+ * - content: A list of content items. Use 'content' to print all content, or
+ * - attributes: HTML attributes for the container element.
+ *
+ * @see template_preprocess_paragraphs_row()
+ *
+ * @ingroup themeable
+ */
+#}
+<div{{ attributes.addClass('paragraph_row') }}>
+  {% if content %}
+    {{- content -}}
+  {% endif %}
+</div>
```

Comment:
```suggestion
<div{{ attributes.addClass('paragraph_row') }}>
  {{- content -}}
</div>
```

Reply:
This was changed in a recent commit https://github.com/SU-SWS/react_paragraphs/pull/59/commits/5a7df0493968f4395f897884883b9a2ca735fbcd#diff-5fac336430ff87026e14e74cd6bc0194

Reply:
What history was I looking at...